### PR TITLE
Move discord embed to use diff

### PR DESCRIPTION
**Pull request by Kapppa**
This should now support discord diff parsing. It should also fix some bugs with the older version as it now supports multiple best and alts.

### CODE REVIEW

File: pb/pb_hooks/embeds.js [hunk 1]
```diff
@@ -5,8 +5,6 @@ module.exports = {
     const DOMAIN = 'https://releases.moe'
 
     const obj = {
-      username: user.get('username'),
```
Comment (ThaUnknown) [inline pb/pb_hooks/embeds.js:8]: no >:(

**Comment by ThaUnknown** [comment]
is this tested, iirc the diff.min.js i generated was full fucked and not compatible with pocketbase

**Comment by Ravencentric** [comment]
planning to test this with https://github.com/ThaUnknown/releases-moe/commit/f692911833e788eae05ad91ea39d816e545efbf1

**Comment by Ravencentric** [comment]
Won't merge till it's tested

File: pb/pb_hooks/embeds.js (hunk 1 above)
Comment (Kapppa) [reply pb/pb_hooks/embeds.js:8]: yes :)

File: pb/pb_hooks/embeds.js [hunk 2]
```diff
@@ -22,36 +20,88 @@ module.exports = {
...
   /**
    * @param {models.Record} record
    * @param {models.Record} user
    */
-  entries (record, user, util) {
+  entries (record, user, util, type) {
     const fields = []
 
+    const preRecord = record.originalCopy()
+    const preRecordExpand = record.originalCopy()
```
Comment (ThaUnknown) [inline pb/pb_hooks/embeds.js:75]: why is this doubled? is this some peak pocketbase bug?

File: pb/pb_hooks/webhook.pb.js [hunk 3]
```diff
@@ -52,3 +55,26 @@ onRecordAfterCreateRequest(e => {
...
+  try {
+    const record = e.record
+    const hooks = $app.dao()?.findRecordsByFilter('hooks', 'event = \'update\'')
+    if (!hooks || !record) return
+    if (record.collection()?.name !== 'torrents') return
+    const store = $app.store()
+    for (const hook of hooks || []) {
+      if (!hook || hook.get('collection') !== 'entries') continue
+
+      store.set(record.get('id'), record.originalCopy())
```
Comment (ThaUnknown) [inline pb/pb_hooks/webhook.pb.js:72]: is this necessary? I feel like doing
```js
const preRecord = record.originalCopy()
const prebest = preRecord.get('isBest')
const postbest = record.get('isBest')
```
accomplishes the same?

Comment (ThaUnknown) [reply pb/pb_hooks/webhook.pb.js:72]: yeah, this is unnecessary, you can simply do inside a `onRecordBeforeUpdateRequest` hook:
```js
    const original = e.record?.originalCopy()
    const updated = e.record?.cleanCopy()
```

**Review by ThaUnknown — APPROVED** [review]

**Review by Kapppa — COMMENTED** [review]
LGTM

**Pull request merged**